web-app/src/app/shared/components/form-field/form-field.component.html

```diff
@@ -146,9 +146,9 @@
   <nz-input-number
     [(ngModel)]="value"
     (ngModelChange)="onChange($event)"
-    [nzMin]="extra.interval_type === 'push' ? 1 : 30"
+    [nzMin]="extra.interval_type === 'push' ? 1 : 10"
     [nzMax]="604800"
-    [nzStep]="extra.interval_type === 'push' ? 1 : 60"
+    [nzStep]="extra.interval_type === 'push' ? 1 : 1"
```

Comment:
```suggestion
    [nzStep]="extra.interval_type === 'push' ? 1 : 60"
```
hi step suggest 60

Reply:
but if min value set 10，and step set 60，will casue 10、70、130…and so on，how about making step 10？

Reply:
hi, min value is 10 but default is 60s, this will casue 120s, 180s... , also user can custom input the number eg: 34, 65

Reply:
hi, that was my mistake, i thought we can not cutsom the input number. The truth is that we can custom set a number bigger than min value (which is 10 now). now i changed the step to 60